gpx-docs/docs/commands/Pat.md

```
@@ -0,0 +1,30 @@
+## Manage PAT for a profile
+
+Use the `pat` command to manage Personal Access Tokens (PAT) securely stored in your operating system's keychain.
+
+#### **NOTE** :
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)